ConsoleView.cpp

```diff
@@ -1,26 +1,41 @@
+//All these functions return void, they are simply here 
+//for the user to call when they want to get information on 
+//a CLASS (or a list of all classes), RELATIONSHIP, or ATTRIBUTE
 
-int UserInputLoop()
-{
-    std::vector<UMLClass> Classes;
-    int count = 0;
-    std::cout << "Enter the amount of Classes to add:" << std::endl;
-    std::cin >> count;
+//Prtint class will be used to print the name of a class given a UMLClass object
+//By using the get_class_name function from the UMLClass, it then prints the name of the Class given
+//@Para: UMLClass OBJ
+PrintClassName(UMLClass X){
+    
+    std::cout<< x.get_class_name(); 
+    
+ }
 
-    std::string ClassName, ClassAttribute;
-    for(int i = 0; i < count; ++i)
-    {
-        std::cout << "Please enter the names of your classes: " << std::endl;
+//PrintClassAttribute is used to print the names of the Attributes in a given UMLClass Object 
+// The attirbutes are stored in a vector so to print out each of the attributes in a class
+// we must iterate over the vector of elements while we print each attirbute stored 
+//the std::for_each takes in two iterators and a function which will be applied 
+//@Para: UMLClass OBJ
+PrintClassAttribute(UMLClass x){
+    auto y= x.get_all_attributes(); //vector created 
+    std::cout << "List of Classes:";
+    std::for_each(y.cbegin(), y.cend(), print); //iteration 
```

Comment:
This line is a bit confusing. I looked up std::foreach and it takes the iterator to the begin and end but the last would be a unary function, so I'm not sure what the 'print' is for

Reply:
thank you, i forgot to define print